src/routes/console/organization-[organization]/billing/planSummary.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+    import { base } from '$app/paths';
+    import { CardGrid, Heading } from '$lib/components';
+    import { Pill } from '$lib/elements';
+    import { Button } from '$lib/elements/forms';
+    import { toLocaleDate } from '$lib/helpers/date';
+    import { daysLeftInTrial, tierFree, tierPro, tierScale } from '$lib/stores/billing';
+    import { organization } from '$lib/stores/organization';
+    import { wizard } from '$lib/stores/wizard';
+    import ChangeOrganizationTierCloud from '$routes/console/changeOrganizationTierCloud.svelte';
+    import { aggregationList } from './store';
+
+    $: currentTier =
+        $organization?.billingPlan === 'tier-2'
+            ? tierScale
+            : $organization?.billingPlan === 'tier-1'
+            ? tierPro
+            : tierFree;
+
+    $: aggregation = $aggregationList.aggregations[0];
+</script>
+
+{#if $organization}
+    <CardGrid>
+        <Heading tag="h2" size="6">Plan summary</Heading>
```

Comment:
```suggestion
        <Heading tag="h2" size="6">Plan</Heading>
```

Reply:
As discussed we'll keep this as "Plan summary" for now.